subsys/bluetooth/mesh/scheduler_internal.h

```diff
@@ -40,10 +47,17 @@ static inline void scheduler_action_unpack(struct net_buf_simple *buf,
 	entry->scene_number = net_buf_simple_pull_le16(buf);
 }
 
-static inline void scheduler_action_pack(struct net_buf_simple *buf,
+static inline bool scheduler_action_pack(struct net_buf_simple *buf,
 				uint8_t idx,
 				const struct bt_mesh_schedule_entry *entry)
 {
+	if (entry->year > MAX_YEAR || entry->day > MAX_DAY ||
+	    entry->hour > MAX_HOUR || entry->minute > MAX_MINUTE ||
+	    entry->second > MAX_SECOND || idx > MAX_IDX ||
+	    entry == NULL) {
```

Comment:
Did you try in your tets to pass entry NULL? Will it cause SEGMENTATION FAULT since all fields will be read as offset from NULL?

Reply:
Now added as part of test_cli_set_prohibited.